### DISCUSSION TITLE
Disable moving Map by touch

### DISCUSSION THREAD

**Discussion by xamix** [mapsforge/vtm#966] [Q&A] [answered]
Hi,

I'm looking to disable moving map by touch only.
I need other touch gesture to be enabled like rotation, pinch to zoom....
I also need to be able to move the map programmatically with animation.

Is there a simple solution to do that?

Regards

**Comment by devemux86** [comment, marked as answer]
Such micromanagement of touch events is not possible.
All touch events can be turned on / off overriding `MapView`'s touch methods.
